src/vuex/modules/wallet.js

```diff
@@ -10,6 +10,7 @@ function getCosmosAddressCreator(bech32Prefix) {
 async function createPolkadotAddress(seedPhrase, addressPrefix) {
   const [{ Keyring }] = await Promise.all([
     import("@polkadot/api"),
+    import("@polkadot/wasm-crypto"),
```

Comment:
is that also needed for signing or only for address creation?

Reply:
for signing it is not needed?

Reply:
Let me check if I get it there too

Reply:
no, signing works just fine (for Schnorrkel accounts).

For Edwards accounts that get "crunched" I still get BadProof though

Reply:
what does crunched mean?

Reply:
Well, in this case I want to mean the addresses that get "destroyed".

Like I had Mario's one that looked like this `JGVLUnieotWLyg25D1uF88EahxmGjc7Dhe2fHUjA5ziBd7i`, but after the Schnorrkel thing it looks like this: `16hApVhutE942rs6G9FrVKbPHjgBANM4qpXmRvC8ENojcxkv` 

And now I cannot sign any transaction with that transformed address. I always get `BadProof`